src/app/reducers/book.spec.ts

```diff
@@ -0,0 +1,158 @@
+import {reducer} from './books';
+import * as fromBooks from './books';
+import {SearchCompleteAction, LoadAction, SelectAction} from '../actions/book';
+import {Book} from '../models/book';
+import {LoadSuccessAction} from '../actions/collection';
+describe('BooksReducer', () => {
+  describe('undefined action', () => {
+    it('should return the default state', () => {
+      const action = {} as any;
```

Comment:
Add some line spacing between the areas of the test where you're arranging setup, performing an action and running assertions